Assets/Scripts/Constants/Types.cs

```diff
@@ -1,5 +1,16 @@
-public class Constants
+public class Types
```

Comment:
Optional: Hm, I don't know if `Types` is the best name for this. Maybe `PizzaTypes` or `PizzaIngredients` or something?